### PR TITLE
Inserted calling of `styling` parameter of `alert` component

### LINKED ISSUE #406: Alert Component: Custom styling is not applied
```
                    alert({
                            background { color { "red" } }
                        }) {
                        content("The custom styling is not applied to the component and it appears in the theme's info color.")
                    }
```
Severity background colors work fine - this is about any styling.

### PR DESCRIPTION
**Pull request by WestHuus**
fixes #406

### CODE REVIEW

**Review by jamowei — CHANGES_REQUESTED** [review]
Do we need a `flexBox` here? When not it's would be nicer in my opinion to use a `div` instead and then no casting is needed to call the `styling()` function ;-)

File: components/src/jsMain/kotlin/dev/fritz2/components/alert.kt [hunk 1]
```diff
@@ -163,7 +164,7 @@ open class AlertComponent : Component<Unit> {
                     }
                 }
 
-                box(baseClass = alertContentCss) {
+                div(baseClass = alertContentCss.name) {
```
Comment (chausknecht) [inline components/src/jsMain/kotlin/dev/fritz2/components/alert.kt:167]: Imho wrong! We need a ``StyleClass`` there and not a ``String`` - as the class also gets applied, I would definitly consider this as false correction!

**Review by chausknecht — CHANGES_REQUESTED** [review]
I think we should cancel this PR in favour of PR #361, which already solves the problem.

File: components/src/jsMain/kotlin/dev/fritz2/components/alert.kt (hunk 1 above)
Comment (chausknecht) [reply components/src/jsMain/kotlin/dev/fritz2/components/alert.kt:167]: Ok, I revoke my comment!

This is for sure something off within our API... the overloaded factories have a different type of course, as the styling DSL (and therefor the type ``StyleClass``) is not available at core level!

**Comment by chausknecht** [comment]
Obsolete, PR #361 covers this already!

**Pull request closed without merging**